modules/transport-netty4/src/main/java/org/elasticsearch/http/netty4/Netty4HttpServerTransport.java

```diff
@@ -311,6 +311,7 @@ protected void initChannel(Channel ch) throws Exception {
             if (handlingSettings.isCompression()) {
                 ch.pipeline().addLast("encoder_compress", new HttpContentCompressor(handlingSettings.getCompressionLevel()));
             }
+            ch.pipeline().addLast("request_creator", new Netty4HttpRequestCreator());
```

Comment:
We can use a constant for this since it's sharable instead of creating a new one for every channel?